src/features/settings/settingsSlice.tsx

```diff
@@ -153,50 +161,48 @@ export const appearanceSlice = createSlice({
   reducers: {
     setFontSizeMultiplier(state, action: PayloadAction<number>) {
       state.appearance.font.fontSizeMultiplier = action.payload;
-
       set(LOCALSTORAGE_KEYS.FONT.FONT_SIZE_MULTIPLIER, action.payload);
     },
     setUseSystemFontSize(state, action: PayloadAction<boolean>) {
       state.appearance.font.useSystemFontSize = action.payload;
-
       set(LOCALSTORAGE_KEYS.FONT.USE_SYSTEM, action.payload);
     },
+    setUserInstanceUrlDisplay(
+      state,
+      action: PayloadAction<InstanceUrlDisplayMode>
+    ) {
+      state.appearance.general.userInstanceUrlDisplay = action.payload;
+      db.setSetting("user_instance_url_display", action.payload);
```

Comment:
how about `show_user_instance`? `user_instance_url_display` is not immediately clear to me

Reply:
I named it that because my plan was to add the ability to turn them off everywhere in the future, so it will be a selector for when the user instance URL should be displayed (always, on posts [the default], never). I can still rename it if you prefer.

Reply:
It's fine, the only reason I wanted to double check is since it's in the db it'll be hard to rename in the future :)